warehouse/query-core/src/main/java/datawave/query/config/ShardQueryConfiguration.java

```diff
@@ -54,174 +53,193 @@
  * Webservice QueryTable and apply them to this configuration object
  */
 public class ShardQueryConfiguration extends GenericQueryConfiguration {
+    public static final String PARAM_VALUE_SEP_STR = new String(new char[] {Constants.PARAM_VALUE_SEP});
     @SuppressWarnings("unused")
     private static final long serialVersionUID = -4354990715046146110L;
     private static final Logger log = Logger.getLogger(ShardQueryConfiguration.class);
-    
-    public static final String PARAM_VALUE_SEP_STR = new String(new char[] {Constants.PARAM_VALUE_SEP});
-    
+    // is this a tld query
+    protected boolean tldQuery = false;
```

Comment:
Can protected be changed to private to guarentee that only accessor methods are used?

Reply:
Can do